src/components/books/book/Book.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {
+  CircularProgressbar,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import './book.css';
```

Comment:
- [ ] [OPTIONAL] Can you kindly remove this line since it's linking to a CSS file that does not exist yet? it's causing some linter errors 😉